test/models/payment_test.rb

```diff
@@ -1,7 +1,29 @@
 require "test_helper"
 
 class PaymentTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  
+  def refund_function
+    @payment = Payment.new(payment_result: @payment_result, refund: true)
```

Comment:
Tests for active_admin workflow are needed as well
- Test should show how do you click this button for refund